frontend/packages/console-app/src/components/modals/restore-pvc/restore-pvc-modal.tsx

```diff
@@ -85,6 +86,11 @@ const RestorePVCModal = withHandlePromise<RestorePVCModalProps>(
       PersistentVolumeClaimKind
     >(PersistentVolumeClaimModel, resource?.spec?.source?.persistentVolumeClaimName, namespace);
 
+    const [scResource, scResourceLoaded, scResourceLoadError] = useK8sGet<StorageClassResourceKind>(
+      StorageClassModel,
+      pvcResource?.spec?.storageClassName,
+    );
+
     const [
```

Comment:
no need to read `VolumeSnapshotClassModel` anymore, u r already reading `StorageClass`, get provisioner from that...

Reply:
https://github.com/openshift/console/pull/13549/files#r1475830560?